codegen/module.go

```diff
@@ -503,7 +503,7 @@ func readPackageInfo(
 		),
 		ExportName:            "New" + qualifiedClassName,
 		QualifiedInstanceName: qualifiedInstanceName,
-		ExportType:            qualifiedInstanceName + qualifiedClassName,
+		ExportType:            qualifiedClassName,
```

Comment:
The instance name is already (or should be) in the importing package name. Compare `bar.Client` with `bar.BarClient`, I think the former is cleaner and reads better, and linting for avoiding stutter also approves the former.